casanovo/config.py

```
@@ -27,185 +27,138 @@
+                        user_cfg_container[new] = user_cfg_container.pop(old)
+                        warning_msg = (
+                            f"Deprecated config option '{old}' "
+                            f"remapped to '{new}'"
+                        )
+                    else:
+                        del user_cfg_container[old]
+                        warning_msg = (
+                            f"Deprecated config option '{old}' "
+                            "is no longer in use"
+                        )
+                    warnings.warn(warning_msg, DeprecationWarning)
```

Ruff (0.15.6)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)